back-office/advanced-parameters/database/advanced_parameters-database-sql_manager.md

```diff
@@ -1,8 +1,59 @@
 # SPECIFICATIONS - ADVANCED PARAMETERS &gt; DATABASE &gt; SQL MANAGER
 
-## Grid
+The SQL manager is a complex feature, which should be reserved for technical people who know how to explore a database using the SQL language. In return for this complexity, it can be extremely powerful and proves immensely useful to those who master it.
 
-Default action in the button: Export Other available actions in the menu: View, Edit, Delete Row action: View
+## 1.7.8 Version 
+The page is going to be migrated into a module. The module will be downloadable for merchants who need it.
```

Comment:
Native module ? Not pre-installed ?

Reply:
It is not defined for now but it seems to be not pre-installed